app/code/Magento/Checkout/Test/Mftf/Test/EndToEndB2CGuestUserTest.xml

```diff
@@ -184,6 +184,194 @@
             <argument name="productVar" value="$$createSimpleProduct2$$"/>
         </actionGroup>
 
+        <comment userInput="Place order with check money order payment" stepKey="commentPlaceOrderWithCheckMoneyOrderPayment" after="guestCheckoutCheckSimpleProduct2InCartItems" />
```

Comment:
Please, revert changes, files in the Checkout module are part of Magento core, not MSI project

Reply:
Reverted changes